src/components/Layout/Sidebar/SidebarLink.tsx

```diff
@@ -71,12 +71,17 @@ export function SidebarLink({
         }
       )}>
       {/* This here needs to be refactored ofc */}
-      <span
-        className={cn({
-          'text-gray-400 dark:text-gray-500': wip,
-        })}>
-        {title}
-      </span>
+      <div>
+        {title}{' '}
+        {canary && (
+          <span
+            title="This feature is available in the latest React Canary"
+            className="ml-2 border border-purple-40 dark:border-purple-50 rounded-full pt-0 pb-0.5 px-2 text-sm text-purple-40 dark:text-purple-50 leading-3">
```

Comment:
I like the purple but was wondering if the colors should be more "traditional" (while fitting our color palette)? I think yellow is typically associated with canary (yellow Chrome icon, yellow Meta app icons, etc).